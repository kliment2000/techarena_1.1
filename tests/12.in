6
10 10 10 10 10 10
5
2 a 5
3 b 2
4 a 5
5 b 2
6 d 2
0
3
2 3 a b
4 5 a b
5 6 b d
