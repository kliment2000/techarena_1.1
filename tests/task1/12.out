(((2 3 {2.a 3.b}) 1) (6 (4 5 {4.a 5.b}) {6.d 5.b})) 2270.0
